src/peer_connection.cpp

```diff
@@ -3024,7 +3061,8 @@ namespace libtorrent {
 				{
 					// when this returns, all outstanding jobs to the
 					// piece are done, and we can restore it, allowing
-					// new requests to it
+					// new req3uests to it
+					auto conn = self();
 					m_disk_thread.async_clear_piece(t->storage(), p.piece
```

Comment:
missing use of lambda with `conn->wrap` here?

Reply:
I started wrapping this call, but this is actually a call on the torrent, so I back-tracked. not all the way though it turns out :P, but maybe I should use the torrent wrap function instead.